src/Interpreters/Aggregator.h

```diff
@@ -282,6 +287,9 @@ class Aggregator final
     /// Get data structure of the result.
     Block getHeader(bool final) const;
 
+    /// Part of a highly experimental functionality - automatic parallel replicas
```

Comment:
I guess it's enough to mention `higly experimental` just in the feature setting description :)